setup.py

```diff
@@ -27,7 +27,16 @@ def get_git_version():
     git_version = check_output('git describe --tags --long --dirty --always'.split()).decode('utf-8').strip()
     return format_version(version=git_version)
 
-def download_ANTF_pulsar_database_file(datadir, version="v1.65"):
+def download_ANTF_pulsar_database_file(datadir, version=None):
```

Comment:
It seems strange that this is needed, since this code depends on `psrqpy`. Doesn't `psrqpy` have the facility for managing the interface to the psrcat database?

Reply:
I hadn't considered that, but yes I think so? @NickSwainston please comment :)

Reply:
We gave this a go because we thought it would speed up the psrqpy queries but it doesn't make much of an improvement. I hoped it would make handling the tests easier if you could tell it which version to use but this was buggy and not really worth it. So might as well remove it and all the places where it is used (ATNF_LOC)

Reply:
Okay, I'll deal with this tomorrow.

Reply:
Okay, I've removed this and associated use. The tests are passing so I assume it is all good.